test/features/cloudnative/codemodules/codemodules.go

```diff
@@ -98,7 +98,7 @@ func InstallFromImage(t *testing.T) features.Feature {
 	builder.Assess("create sample namespace", sampleApp.InstallNamespace())
 
 	// Register dynakube install
-	dynakube.Install(builder, helpers.LevelAssess, &secretConfigs[0], cloudNativeDynakube)
+	dynakube.Install(builder, helpers.LevelAssess, &secretConfigs[0], cloudNativeDynakube, false)
```

Comment:
I don't understand this change, if we never use `true`, why do we need it?

Reply:
it is used in the release e2e test, where I check if we use a old version which does not know v1beta2

Reply:
I am not sure how to do that differently. In the case we use a version that does not know v1beta2 we convert our dynakube to v1beta1 and use v1beta1 then instead of v1beta2